src/main/java/codeu/controller/ConversationServlet.java

```diff
@@ -131,9 +132,8 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
 
     conversationStore.addConversation(conversation);
 
-    // set type of activity to 2 (new conversation) and add to activity list
-    conversation.setType(2);
-    activityStore.addActivity(conversation);
+    // create activity with type NEW_CONVERSATION and add to activity list
+    activityStore.addActivity(new Activity(Activity.Type.NEW_CONVERSATION, conversation.getId(), conversation.getCreationTime(), UUID.randomUUID()));
```

Comment:
As a good convention, keep the column length (no. of characters on each line) restricted to 100: https://google.github.io/styleguide/javaguide.html#s4.4-column-limit

Here & everywhere. This might help: https://github.com/google/google-java-format